README.bazel.md

```diff
@@ -8,6 +8,16 @@ This repository uses Bazel to provide a monorepo developer experience.
 - Run `bazel run format path/to/file` to re-format a single file.
 - Run `pre-commit install` to auto-format changed files on `git commit`; see https://pre-commit.com/.
 
+## Linting code
+
+- Run `bazel lint //...` or any other target pattern to check for lint violations.
+
+> Note: the `lint` command is provided by Aspect CLI but is *not* part of the Bazel CLI provided by Google.
+> Aspect CLI makes the developer experience for linting a lot nicer, but it's not required.
+> At its core, linting is powered by a `bazel build --aspects=[linting aspects...]` command.
```

Comment:
Ideally right here we would have the Bazel command or commands needed to use the aspects feature to run linting without the Aspect CLI:

* demonstrates the value of the Aspect CLI.
* demonstrates how to use aspects

I see the link further below that has a full length example script to do it, but it would be nice to show the Bazel parts without getting lost in bash command line parsing etc. complexity.

Reply:
Yeah you're right, we can give something more helpful and instructive here, PTAL